src/components/ErrorMessagePanel/ErrorMessagePanel.js

```diff
@@ -0,0 +1,24 @@
+import { Typography, makeStyles } from "@material-ui/core"
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    margin: 0,
+    color: theme.palette.error.main,
+  },
+}))
+
+const ErrorMessagePanel = ({ errors }) => {
```

Comment:
The only prop this component needs is the error message to display. Also, please be mindful about you naming, this is not a panel, it's just an error message, so calling this component something like`InputError` would be more appropriate